app/models/journeys/additional_payments_for_teaching.rb

```diff
@@ -30,24 +30,43 @@ module AdditionalPaymentsForTeaching
       }
     }.freeze
 
-    def final_policy_year(policy)
-      {
-        Policies::EarlyCareerPayments => EligibilityCheckable::FINAL_COMBINED_ECP_AND_LUP_POLICY_YEAR,
-        Policies::LevellingUpPremiumPayments => EligibilityCheckable::FINAL_LUP_POLICY_YEAR
-      }[policy]
-    end
-
     def set_a_reminder?(itt_academic_year:, policy:)
       policy_year = configuration.current_academic_year
-      return false if policy_year >= final_policy_year(policy)
+      return false if policy_year >= policy::POLICY_END_YEAR
 
       next_year = policy_year + 1
-      eligible_itt_years = JourneySubjectEligibilityChecker.selectable_itt_years_for_claim_year(next_year)
+      eligible_itt_years = selectable_itt_years_for_claim_year(next_year)
       eligible_itt_years.include?(itt_academic_year)
     end
 
     def requires_student_loan_details?
       true
     end
+
+    def selectable_subject_symbols(journey_session)
```

Comment:
Suggestion: this could probably live in the `EligibleIttSubjectForm` class.

Reply:
👍 that's nicer, will do this as a separate pr though

Reply:
https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/3461